server/config/routes/scripts.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+
+import { saveNewScript,
+         getAllScripts,
+         getScriptById,
+         updateScriptById,
+         addQuestionToScript,
+         getQuestionsByScriptId,
+         deleteScriptById } from '../../controllers/scripts';
+
+const router = express.Router();
+
+router.route('/').post(saveNewScript);
+router.route('/').get(getAllScripts);
+router.route('/:id').get(getScriptById);
+router.route('/:id').put(updateScriptById);
+router.route('/:id').delete(deleteScriptById);
+router.route('/addQuestionToScript/:id/:questionId/:sequenceNum').post(addQuestionToScript);
```

Comment:
Would consider changing the route for this and the one below to `:id/questions` and make it accept form params on POST for adding a question and a plain GET to get questions. Makes for a cleaner routespace for our server.